mmdeploy/backend/sdk/export_info.py

```diff
@@ -141,9 +141,53 @@ def get_inference_info(deploy_cfg: mmengine.Config, model_cfg: mmengine.Config,
 def get_preprocess(deploy_cfg: mmengine.Config, model_cfg: mmengine.Config):
     task_processor = build_task_processor(
         model_cfg=model_cfg, deploy_cfg=deploy_cfg, device='cpu')
-    transforms = task_processor.get_preprocess()
-    assert transforms[0]['type'] == 'LoadImageFromFile', 'The first item'\
-        ' type of pipeline should be LoadImageFromFile'
+    pipeline = task_processor.get_preprocess()
```

Comment:
Should this part be moved to `get_preprocess` of each Task class? @AllentDan

Reply:
Yes. Preprocess and post process were moved to codebases.